kopf/reactor/causation.py

```diff
@@ -67,6 +67,7 @@ class Cause(NamedTuple):
     initial: bool
     body: MutableMapping
     patch: MutableMapping
+    digest: Union[str, int]
```

Comment:
in what conditions the `digest` can be `int`?

Reply:
When it is reimplemented to something else (e.g. crc32). We just show that we do not care that it is necessary a string.

Reply:
I see, so in the specification we don't care wether this string or int.